packages/x-components/src/x-modules/history-queries/components/my-history.vue

```diff
@@ -1,58 +1,78 @@
 <template>
-  <BaseSuggestions
-    :suggestions="historyQueries"
-    class="x-my-history-queries"
-    data-test="my-history-queries"
-    :animation="animation"
-  >
-    <template #default="{ suggestion, index }">
-      <!--
+  <ul v-if="historyQueries.length > 0" class="x-my-history x-list">
+    <li
+      v-for="(list, date) in historyQueriesGroupByDate"
+      :key="date"
+      class="x-my-history-item"
+      data-test="my-history-item"
+    >
+      <span class="x-my-history-item__date" data-test="my-history-date">{{ date }}</span>
+      <BaseSuggestions
+        :suggestions="list"
+        class="x-my-history-queries"
+        data-test="my-history-queries"
+        :animation="animation"
+      >
+        <template #default="{ suggestion, index }">
+          <!--
         @slot History Query item
             @binding {Suggestion} suggestion - History Query suggestion data
             @binding {number} index - History Query suggestion index
       -->
-      <slot name="suggestion" v-bind="{ suggestion, index }">
-        <HistoryQuery
-          :suggestion="suggestion"
-          data-test="history-query-item"
-          class="x-history-queries__item"
-        >
-          <template #default>
-            <!-- eslint-disable max-len -->
-            <!--
+          <slot name="suggestion" v-bind="{ suggestion, index }">
+            <HistoryQuery
+              :suggestion="suggestion"
+              data-test="history-query-item"
+              class="x-history-queries__item"
+            >
+              <template #default>
+                <!-- eslint-disable max-len -->
+                <!--
               @slot History Query content
                   @binding {Suggestion} suggestion - History Query suggestion data
                   @binding {number} index - History Query suggestion index
             -->
-            <!-- eslint-enable max-len -->
-            <slot name="suggestion-content" v-bind="{ suggestion, index }" />
-          </template>
-          <template #remove-button-content>
-            <!--
+                <!-- eslint-enable max-len -->
+                <slot name="suggestion-content" v-bind="{ suggestion, index }">
+                  <HistoryTiny class="x-icon--l" />
```

Comment:
Even if it is the standard implementation, do we need to add this icon here? Then it will always be included in the bundle, even if you change it.